### PR TITLE
add submit messages & left room

### PR DESCRIPTION
**Pull request by Andrij-Maksymyshyn**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -30,20 +29,63 @@ io.on("connection", socket => {
   socket.on("join", ({ name, room }) => {
     socket.join(room);
 
-    const { user } = addUser({ name, room });
+    const { user, isUserExist } = addUser({ name, room });
+    // console.log("isUserExist", isUserExist);
```
Comment (chestnyh) [inline server/index.js]: No need for this comment

File: README.md [hunk 2]
```diff
@@ -1,30 +1,53 @@
...
 
-- 🔭 I’m currently working on [real-time-chat](https://github.com/Andrij-Maksymyshyn/real-time-chat)
+### 🏠 [Homepage](https://github.com/Andrij-Maksymyshyn/real-time-chat)
 
-- 🌱 I’m currently learning **Node.js**
+## Install
 
-- 🤝 I’m looking for help with [Oleksij Chestnyh (mentor)](https://github.com/chestnyh)
+```sh
+yarn
```
Comment (chestnyh) [inline README.md]: yarn from project doesn't work. You should describe here from what folder you should run in

File: README.md [hunk 3]
```diff
@@ -1,30 +1,53 @@
...
+```
 
-- 👨‍💻 All of my projects are available at [https://github.com/Andrij-Maksymyshyn](https://github.com/Andrij-Maksymyshyn)
+## Usage
 
-- 💬 Ask me about **react.js, Node.js, socket.io**
+```sh
+yarn start
```
Comment (chestnyh) [inline README.md]: This doesn't work from root folder

File: README.md [hunk 4]
```diff
@@ -1,30 +1,53 @@
...
 
-- 📫 How to reach me **maxandrij@ukr.net**
+## Development
 
-<h3 align="left">Connect with me:</h3>
-<p align="left">
-<a href="https://linkedin.com/in/https://www.linkedin.com/in/andrewmax777/" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="https://www.linkedin.com/in/andrewmax777/" height="30" width="40" /></a>
-</p>
+```sh
+yarn dev
```
Comment (chestnyh) [inline README.md]: Doesn't work from root

File: README.md (hunk 2 above)
Comment (Andrij-Maksymyshyn) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (Andrij-Maksymyshyn) [reply README.md]: fixed

File: server/index.js (hunk 1 above)
Comment (Andrij-Maksymyshyn) [reply server/index.js]: fixed

File: README.md (hunk 4 above)
Comment (Andrij-Maksymyshyn) [reply README.md]: fixed

File: README.md [hunk 5]
```diff
@@ -1,30 +1,53 @@
...
 
-- 🔭 I’m currently working on [real-time-chat](https://github.com/Andrij-Maksymyshyn/real-time-chat)
+### 🏠 [Homepage](https://github.com/Andrij-Maksymyshyn/real-time-chat)
 
-- 🌱 I’m currently learning **Node.js**
+## Install
 
-- 🤝 I’m looking for help with [Oleksij Chestnyh (mentor)](https://github.com/chestnyh)
+```sh
+cd ./server yarn
```
Comment (chestnyh) [inline README.md:11]: This doesn't work for me. Got the error above:
```
 cd ./server yarn
cd: string not in pwd: ./server
```
Does it work for you?

**Review by chestnyh — APPROVED** [review]

**Pull request merged**